docs/README.md

```diff
@@ -50,12 +50,12 @@ The next level is a folder. It can contain nesting folders and `*.md` files. Eac
 - group - a regular group with a title and list of items
 - single - good for groups with a single item
 - hidden - the folder won't be shown in the sidebar but when you open a page from this group sidebar is present.
-- plugins - the "special" group with is generated not from the `*.md` files located in the content folder, but from README.md files from plugin packages
+- plugins - the "special" group which is generated not from the `*.md` files located in the content folder, but from README.md files from plugin packages
```

Comment:
```suggestion
- plugins - the "special" group that is generated, not from the `*.md` files located in the content folder, but from README.md files located in plugin packages
```

I think we should use `that` over `which` in this case because the subclause is restrictive, it can't be dropped without altering the meaning of the sentence.